pkgs/by-name/su/surreal-engine/package.nix

```
@@ -0,0 +1,71 @@
+{
+  alsa-lib,
+  cmake,
+  dbus,
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  openal,
+  pkg-config,
+  SDL2,
+  stdenv,
+  vulkan-loader,
+  waylandpp,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "surreal-engine";
+  version = "2024-08-31";
```

https://github.com/NixOS/nixpkgs/blob/51031f223c0f05491ae3e24d771ebc76821b6ae1/pkgs/README.md#versioning

```suggestion
  version = "0-unstable-2024-08-31";
```